packages/xo-web/src/common/intl/index.js

```diff
@@ -2,7 +2,7 @@ import moment from 'moment'
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import { FormattedMessage, IntlProvider as IntlProvider_ } from 'react-intl'
+import { FormattedMessage, IntlProvider as OriginalIntlProvider } from 'react-intl'
```

Comment:
why renaming it ?

Reply:
Ok, as an exception because they are a lot of linting issue in xo-web 5, let's decide that Lint fix in this specific package is allowed in any PRs.

@ByScripts @fbeauchamp @MathieuRA @ByScripts Everybody agree?